examples/sugarscape_g1mt/run.py

```diff
@@ -0,0 +1,101 @@
+import sys
+import pandas as pd
+import matplotlib.pyplot as plt
+import networkx as nx
+import mesa
+from sugarscape_g1mt.model import SugarscapeG1mt
+from sugarscape_g1mt.server import server
+
+
+#Analysis
```

Comment:
This needs to be `# Analysis`. It means the file hasn't been Black-ed yet.

Reply:
Hmmm...not sure why black didn't run, but it is fixed now